packages/utils/src/facility/sync-facilities.ts

```diff
@@ -0,0 +1,171 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { Facility } from "@metriport/api-sdk/medical/models/facility";
+import { FacilityInternalDetails, FacilityType } from "@metriport/core/domain/facility";
+import { sleep } from "@metriport/shared";
+import { getEnvVarOrFail } from "@metriport/shared/common/env-var";
+import axios from "axios";
+import dayjs from "dayjs";
+import { getCqFacilitySafe, getCwFacilitySafe, getFacilityByNpi, readNpisFromCsv } from "./utils";
+import { Command } from "commander";
+
+/*
+ * This script will read NPIs from a local csv.
+ * It will try to sync the facility with CommonWell and CareQuality if it's not already there.
+ *
+ * It console logs the result of processing.
+ *
+ * Format of the .csv file:
+ * - first line contains column name: npi
+ * - minimum columns: npi
+ *
+ * Set the API_URL env var.
+ * Set the input variables below.
+ *
+ * Execute this with:
+ * $ ts-node src/facility/sync-facilities --input-path <inputpath> --cx-id <cxId>
+ */
+interface FacilitySyncParams {
+  inputPath: string;
+  cxId: string;
+}
+
+const internalUrl = getEnvVarOrFail("API_URL");
+const defaultActive = true;
+const defaultType = FacilityType.initiatorAndResponder;
+const timeout = dayjs.duration(0.5, "seconds"); //There is 2 sleeps, one at the start of the loop and one in the middle of the loop.
+
+async function main({ inputPath, cxId }: FacilitySyncParams) {
+  const npis = await readNpisFromCsv(inputPath);
+  const cwFound: string[] = [];
+  const cwNotFound: string[] = [];
+  const cqFound: string[] = [];
+  const cqNotFound: string[] = [];
+  const noOid: string[] = [];
+  const facilityNotFound: string[] = [];
+  const synced: string[] = [];
+  const syncFailed: string[] = [];
+
+  for (const npi of npis) {
+    await sleep(timeout.asMilliseconds());
+    console.log(`Processing facility: ${npi}`);
+    let facility: Facility | null = null;
+    try {
+      facility = await getFacilityByNpi(cxId, npi);
+    } catch (error) {
+      facilityNotFound.push(npi);
+      console.log(error);
+      continue;
+    }
+    if (!facility) {
+      console.log(`Facility not found: ${npi}`);
+      facilityNotFound.push(npi);
+      continue;
+    }
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const facilityOid = (facility as any).oid;
+    if (!facilityOid) {
+      console.log(`Facility has no OID: ${npi}`);
+      noOid.push(npi);
+      continue;
+    }
+
+    const cwOrg = await getCwFacilitySafe(cxId, facility.id, facilityOid);
+    const cqOrg = await getCqFacilitySafe(cxId, facility.id, facilityOid);
+
+    if (cwOrg) cwFound.push(npi);
+    else cwNotFound.push(npi);
+
+    if (cqOrg) cqFound.push(npi);
+    else cqNotFound.push(npi);
+
+    if (cwOrg && cqOrg && cwOrg.active && cqOrg.active) {
+      console.log(`Facility is active in both CW and CQ`);
+      continue;
+    }
```

Comment:
We should check if active on cq/cw is diff from the desired state and update it, not just return if active (what if the goal is to deactivate?)